docs/websites/nginx/nginx-ssl-and-tls-deployment-best-practices.md

```diff
@@ -0,0 +1,286 @@
+---
+author: 
+  name: Linode Community
+  email: docs@linode.com
+description: 'This guide details best practices for deploying SSL and TLS in conjunction with nginx.'
+keywords: 'nginx,ssl,tls'
+license: '[CC BY-ND 4.0](http://creativecommons.org/licenses/by-nd/4.0)'
+published: 'Monday, July 11th, 2016'
+modified: Monday, July 11th, 2016
+modified_by:
+  name: Linode
+title: 'Nginx SSL and TLS Deployment Best Practices'
+contributor:
+  name: Ryan Laverdiere
+  link: https://github.com/capecodrailfan
+external_resources:
+- '[Qualys SSL Labs SSL Server Test](https://www.ssllabs.com/ssltest/)'
+- '[KeyCDN HTTP/2 Test](https://tools.keycdn.com/http2-test)'
+---
+
+*This is a Linode Community guide. [Write for us](/docs/contribute) and earn $250 per published guide.*
+<hr>
+
+This guide is intended to inform you of some additional configuration options that nginx uses when serving HTTPS. While these features help optimize nginx for SSL and TLS, this is by no means a complete guide to securing nginx or your Linode. The best way to ensure your server remains secure is to not only to configure it properly, but to follow best security practices at all times. This guide is intended to be one of many steps toward creating the most secure environment possible. 
```

Comment:
Proof note: ...is to not only to configure it properly... = ...is to not only configure it properly...